src/Gallery/GalleryItemSettings.tsx

```diff
@@ -1,17 +1,18 @@
 import React, { FC, useState, useEffect } from 'react';
-import { WhpptButton, WhpptInput, WhpptDayInput } from '../ui/components';
+import { WhpptButton, WhpptInput, WhpptDayInput, WhpptIcon } from '../ui/components';
 import { WhpptGalleryTag } from './GalleryTag';
 import { useWhppt } from '../Context';
 import { GalleryItem } from './Model';
 import { toast } from 'react-toastify';
 
 type GalleryItemSettingsProps = {
-  use: () => void;
+  use: (item: GalleryItem) => void;
   selectedId: string;
   remove: (id: string) => void;
+  setSettingsOpen: (open: boolean) => void;
 };
 
-export const GalleryItemSettings: FC<GalleryItemSettingsProps> = ({ use, selectedId, remove }) => {
+export const GalleryItemSettings: FC<GalleryItemSettingsProps> = ({ use, selectedId, remove, setSettingsOpen }) => {
```

Comment:
This took me a while to understand why the settings panel would have to open. Could we make this prop read close. The on the calling side we can still call setSettingsOpen(false)

Reply:
done